tests/a-framework/CMakeLists.txt

```diff
@@ -10,14 +10,23 @@ SET(TEST_TARGET_DEBUG dummy.debug)
 DEAL_II_PICKUP_TESTS()
 
 #
-# Limit concurrency between the two parameter file tests:
+# Limit concurrency between the two parameter file tests.
 #
-FOREACH(_build ${DEAL_II_BUILD_TYPES})
-  STRING(TOLOWER ${_build} _build)
-  SET_TESTS_PROPERTIES(a-framework/parameter_file_2.${_build} PROPERTIES
-    DEPENDS a-framework/parameter_file_1.${_build}
-    )
-ENDFOREACH()
+# We only do this for CMake-3.4 or newer that has policy CMP0064 and
+# supports the IF(TEST ...) operator.
+#
+IF(POLICY CMP0064)
+  CMAKE_POLICY(SET CMP0064 NEW)
+  FOREACH(_build ${DEAL_II_BUILD_TYPES})
+    IF( TEST a-framework/parameter_file_2.${_build} AND
```

Comment:
this does not need to be tolower?

Reply:
(you fixed this when I wrote the comment)

Reply:
Jup. Still verifying that this does the right thing in all cases.

Reply:
Done. Looks good on my machine now.